src/components/AllProducts.jsx

```diff
@@ -17,10 +17,13 @@ const AllProducts = () => {
 
 	return (
 		<div className="allProducts">
+			{' '}
+			<h1>Welcome to Grace Shopper</h1>
+			<h2>The place to buy your products!</h2>
 			<h1>Products:</h1>
 			<div>
-				{products.length &&
-					products.map((product) => {
+				{!!allProducts.length &&
```

Comment:
it's good to either use !! or `allProducts.length > 0`. otherwise you may get a bug, or you may print a zero.

but in this case, you don't need `allProducts.length` at all.